packages/sdk-js/src/index.ts

```diff
@@ -668,8 +682,10 @@ class Client extends BaseClient {
   }
 }
 
-// Export types for easier use
-export * from "./types";
+export { Environment, WorkflowStatus, TriggerType } from "@avaprotocol/types";
+
+export { ExecutionStatus };
```

Comment:
Review all these exports. If they are already defined in the @avaprotocol/types, then no need to export them here, or from the src/index.ts.

The exports were here because previously types are defined in this file, or the already deleted types.tx, but not from the packages/types package.